tests/test_functions.py

```diff
@@ -11,6 +11,12 @@ def test_compound_interest():
     assert round(functions.compound_interest(100, 0.10, 10, 12), 2) == 270.70
 
 
+def test_effective_interest():
```

Comment:
May want to add a 0% interest rate test, looks like it will work but good for avoiding divide by 0 errors in the future